ScreenManager.lua

```diff
@@ -76,8 +136,11 @@ end
 -- @param ...    (vararg) One or multiple arguments passed to the new screen.
 --
 function ScreenManager.switch( screen, ... )
-    clear();
-    ScreenManager.push( screen, ... );
+    validScreen( screen );
```

Comment:
We could use a single variable and define an action:
`change = { action = 'switch', args = { ... }, screen = screen }`